lib/rouge/lexers/syzlang.rb

```diff
@@ -0,0 +1,315 @@
+        rule %r/[\(\){}\[\];,]/, Punctuation
+        rule inline_spaces, Text
+        rule %r//, Text, :pop!
+      end
+
+      state :resource_name do
+        mixin :mixin_name
+        rule %r//, Text, :pop!
+      end
+
+      state :resource_type do
+        rule %r/\[/, Punctuation, :arg_type
+        rule %r/\]/, Punctuation, :pop!
+      end
+
+      state :resource_values do
+        rule %r/:/ do
+          token Punctuation
+          push :resource_values_list
+          push :space_break
+        end
+        rule %r//, Text, :pop!
+      end
+
+      state :resource_values_list do
+        rule inline_spaces, Text
+        mixin :mixin_name
+        mixin :mixin_number
+        mixin :mixin_string
+        rule %r/,/, Punctuation, :space_break
+        rule %r//, Text, :pop!
+      end
+
+      state :flags_list do
+        rule inline_spaces, Text
+        mixin :mixin_name
+        mixin :mixin_number
+        mixin :mixin_string
+        rule %r/,/, Punctuation, :space_break
+        rule %r//, Punctuation, :pop!
+      end
+
+      state :syscall_args do
+        rule spaces, Text
+        rule comment, Comment
+        rule id do
+          token Name
+          push :arg_type
+          push :space_break
+        end
+        rule %r/,/, Punctuation
+        rule %r/\)/, Punctuation, :pop!
+      end
+
+      state :syscall_retval do
+        mixin :mixin_name
+        rule %r//, Text, :pop!
+      end
+
+      state :syscall_mods do
+        rule %r/\(/, Punctuation, :term_list
+        rule %r//, Text, :pop!
+      end
+
+      state :struct_fields do
+        rule id do
+	  token Name
+          push :space_break
+          push :struct_field_mods
+          push :inline_break
+          push :arg_type
+          push :space_break
+        end
+        rule %r/[}\]]/, Punctuation, :pop!
+      end
+
+      state :struct_field_mods do
+        rule %r/\(/, Punctuation, :term_list
+        rule %r//, Text, :pop!
+      end
+
+      state :struct_mods do
+        rule %r/\[/, Punctuation, :term_list
+        rule %r//, Text, :pop!
+      end
+
+      state :type_name do
+        mixin :mixin_name
+        rule %r//, Text, :pop!
+      end
+
+      state :type_args do
+        rule %r/\[/, Punctuation, :type_args_list
+        rule %r//, Text, :pop!
+      end
+
+      state :type_args_list do
+        rule spaces, Text
+        rule comment, Comment
+        mixin :mixin_name
+        rule %r/,/, Punctuation
+        rule %r/\]/, Punctuation, :pop!
+      end
+
+      state :type_body do
+        rule %r/[{\[]/ do
+          token Punctuation
+          pop!
+          push :space_break
+          push :struct_mods
+          push :inline_break
+          push :struct_fields
+          push :space_break
+        end
+        rule %r// do
+          pop!
+          push :arg_type
+        end
+      end
+
+      state :root do
+        # Whitespace.
+        rule %r/#{spaces}/, Text
```

Comment:
**suggestion:** Can we use simplify this statement? 🤔 

```suggestion
        rule spaces, Text
```

Reply:
Indeed, done!